### PR TITLE
Incorporating SimpleAnalytics to gather basic user information.

### PR DESCRIPTION
**Pull request by sarahkeh**
## Description

Added script tags to incorporate SimpleAnalytics.  The script tag can be included conditionally, default on (true).  Sites need to be added to SimpleAnalytics account and team members can be added from there to view and use the data.

Simple Analytics is a lightweight user tracker that doesn't rely on cookies.  Some ad blockers still block this site, so if data isn't appearing, check that the code isn't getting blocked there.  

Jira: CV2-743

## Type of change

- [ ] Performance improvement and/or refactoring (non-breaking change that keeps existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Security mitigation or enhancement
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Automated test (add or update automated tests)

## How has this been tested?

End to end testing, added temporary external localhost and added the site to the SimpleAnalytics app to confirm the data was being collected

## Things to pay attention to during code review

Check that an ad blocker isn't blocking the analytics. config-server contains the useAnalytics tag that can turn the script tag on and off. 

## Checklist

- [X] I have performed a self-review of my own code
- [X] I've made sure my branch is runnable and given good testing steps in the PR description
- [X] I considered secure coding practices when writing this code. Any security concerns are noted above.
- [ ] I have commented my code in hard-to-understand areas, if any
- [ ] I have made needed changes to the README
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I implemented any new components, they are self-contained, their `propTypes` are declared and they use React Hooks and, if data-fetching is required, they use Relay Modern with fragment containers
- [ ] If my components involve user interaction - specifically button, text fields, or other inputs - I have added a [BEM-like class name](https://meedan.atlassian.net/wiki/spaces/ENG/pages/1327628289/Naming+conventions+for+interactive+elements) to the element that is interacted with
- [ ] To the best of my knowledge, any new styles are applied according to the design system
- [ ] If I added a new external dependency, I included a rationale for doing so and an estimate of the change in bundle size (e.g., checked in https://bundlephobia.com/)

### CODE REVIEW

**Review by caiosba — APPROVED** [review]
First Meedan PR for Sarah, thanks! :champagne:

File: src/web/views/index.js [hunk 1]
```diff
@@ -41,6 +41,13 @@ const heapAnalytics = (config) => {
     </script>` : '';
 };
 
+const simpleAnalytics = (config) => {
+  return config.useAnalytics ?
+    `<script async defer data-collect-dnt="true" src="https://scripts.simpleanalyticscdn.com/latest.js"></script>
```
Comment (dariusk) [inline src/web/views/index.js:46]: The `data-collect-dnt="true"` means that if a user has "Do Not Track" enabled clientside, we will ignore it and collect data from them anyway. Is this something we want to do? CC @huslage

**Comment by caiosba** [comment]
Works for me locally:

Captura de tela de 2024-03-12 16-36-02

**Review by dariusk — APPROVED** [review]

**Comment by brianfleming** [comment]
Random question, are we going to deprecate Heap? or do we still have use for that in another way?

**Comment by caiosba** [comment]
> Random question, are we going to deprecate Heap? or do we still have use for that in another way?

Both Heap and SimpleAnalytics are handled by configuration keys, so it will be up to us... but I think we should try to use only one in production.

**Comment by brianfleming** [comment]
> > Random question, are we going to deprecate Heap? or do we still have use for that in another way?
> 
> Both Heap and SimpleAnalytics are handled by configuration keys, so it will be up to us... but I think we should try to use only one in production.

agreed, if we are not actively using Heap for anything, and this is the replacement, lets ensure we only have this in prod

**Pull request merged**